docs/pages/guides/using-custom-fonts.mdx

```diff
@@ -69,7 +69,7 @@ Then, you can use the font on the `Text` component by using `fontFamily` style p
 <Text style={{ fontFamily: 'Inter-Black', fontSize: 30 }}>Inter Black</Text>
 ```
 
-Alternatively, you can use [`Fonts.loadAsync`](#using--fontloadasync--instead-of-the) to load the fonts in your app.
+Alternatively, you can use [`Font.loadAsync`](#using-fontloadasync-instead-of-the) to load the fonts in your app.
```

Comment:
Thank you for addressing this Kim! 👍 

```suggestion
Alternatively, you can use [`Font.loadAsync`](#using-fontloadasync-instead-of-the-usefonts-hook) to load the fonts in your app.
```

I have run locally and it looks like this is the correct hash link for the section.

Reply:
Good catch, not sure what happened there.